README.md

```diff
@@ -44,19 +44,19 @@ You can choose between using the installer or using the portable build. Using th
 
 * **Debian-based**:
 
-  * `sudo apt install libgdk-pixbuf-2.0-0 libgl1 libglu1-mesa libglvnd0 libgtk-3-0 libusb-0.1-4 libxinerama1 libxtst6`
+  * `sudo apt install libgdk-pixbuf-2.0-0 libgl1 libglvnd0 libgtk-3-0 libusb-0.1-4 libxinerama1 libxtst6`
```

Comment:
I think this PR is incomplete and this change should be reverted. [The Ubuntu pipeline fails](https://github.com/florczakraf/itgmania/actions/runs/10357494995/job/28669628836#step:6:11) when `libglu1-mesa-dev` is removed because we still try to build `glew` from externals, which by default utilizes `glu` and in turn would require the runtime library. There seem to be an option `GLEW_NO_GLU` to disable it, but I haven't tested whether that's enough to let it build and if the game itself would work without those extensions

Reply:
> I think this PR is incomplete and this change should be reverted. [The Ubuntu pipeline fails](https://github.com/florczakraf/itgmania/actions/runs/10357494995/job/28669628836#step:6:11) when `libglu1-mesa-dev` is removed because we still try to build `glew` from externals, which by default utilizes `glu` and in turn would require the runtime library. There seem to be an option `GLEW_NO_GLU` to disable it, but I haven't tested whether that's enough to let it build and if the game itself would work without those extensions

The referenced instructions are for users trying to run the binary. It is accurate that with this change, one can run the compiled binary without `libGLU.so.1` installed on their system. However, the dependency is still needed for building, which is why I didn't touch `ci.yml`.

Reply:
the executable links against libglu so it's a runtime requirement: 
```
florczak-> git describe --tags --dirty
v0.9.0-54-ga0e4f6a4b0

florczak-> ldd itgmania | grep -i glu
	libGLU.so.1 => /lib/x86_64-linux-gnu/libGLU.so.1 (0x00007fac2761f000)

florczak-> sudo mv /lib/x86_64-linux-gnu/libGLU.so.1 /lib/x86_64-linux-gnu/libGLU.so.1.old

florczak-> ./itgmania
./itgmania: error while loading shared libraries: libGLU.so.1: cannot open shared object file: No such file or directory
```

Reply:
Can you elaborate on how you built it & what distro you are on so I can replicate what you're seeing?

This is what I did on both Fedora and OpenSUSE:

1. Clone the `beta` branch, without `libGLU1` or `libglu1-mesa-dev` installed.
    -  If you try to build the game without `libglu1-mesa-dev`, it should fail on  `RageDisplay_OGL.cpp`.
3. Apply this patch.
4. Install `libglu1-mesa-dev`.
5. Build the game.
6. Remove `libglu1-mesa-dev`. Make sure `libGLU1` is not installed.
7. Launch the game. It should run.

Reply:
> Can you elaborate on how you built it & what distro you are on so I can replicate what you're seeing?

It doesn't matter, the dynamic linker works the same way everywhere. I've shown you in the snippet above that the newly built executable depends on the libglu shared library. I could stop here because I've shared everything needed to prove that.

However, I've created https://github.com/florczakraf/itgmania/commit/ffd431d3457315b2a4a3ffd2fd02250b26740d39 on top of `beta` to reproduce the problem [in github pipeline](https://github.com/florczakraf/itgmania/actions/runs/10377215851/job/28730921401#step:7:1) so there's no wiggle room. Here are the extra steps appended to our normal build to follow your steps:
```
      - name: List runtime libraries
        run: ldd ./itgmania
      - name: remove libglu
        run: sudo apt-get remove -y libglu1-mesa-dev libglu1-mesa
      - name: List runtime libraries again
        run: ldd ./itgmania
      - name: Try to run
        run: ./itgmania
```

1. We're on top of beta (a0e4f6a4b07cd5af040c1b118977ea2d93966f46)
2. This PR has been merged there already
3. Github pipeline installs `libglu1-mesa-dev` (unaltered)
4. Github pipeline does the usual linux build (unaltered)
5. `libglu1-mesa-dev` and `libglu1-mesa` are uninstalled in `remove libglu` step
6. running the game fails in `Try to run` step as expected because of the missing library (`./itgmania: error while loading shared libraries: libGLU.so.1: cannot open shared object file: No such file or directory`

Between 4-5 and 5-6 there are also a dumps of the runtime requirements for `itgmania` binary in the pipeline so you can see that it depends on `libGLU.so.1` and it's available:
```
	libGLU.so.1 => /lib/x86_64-linux-gnu/libGLU.so.1 (0x00007fcbb9bae000)
```
After uninstalling libglu in 5. `ldd` reports that it's not available:
```
	libGLU.so.1 => not found
```
and running the game fails because of it:
```
Run ./itgmania
./itgmania: error while loading shared libraries: libGLU.so.1: cannot open shared object file: No such file or directory
```
If this library was available, we'd expect the game to fail on creating a window, finding audio device or something else caused by the limited pipeline environment, but we've never got to executing our code, because the app startup fails in dynamic linker.